sagemaker-inference-recommender/sklearn-inference-recommender/skleran-inference-recommender.ipynb

```diff
@@ -0,0 +1,715 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        ModelPackageGroupDescription="My sample sklearn model package group",
</pre><p>"sklearn" is pretty generic - add a couple words about regression, binary classification, or California housing so the customer would remember what this model package group is from if they find it later?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>